poetry/masonry/builders/builder.py

```diff
@@ -75,7 +75,9 @@ def build(self):
     @lru_cache(maxsize=None)
     def find_excluded_files(self):  # type: () -> Set[str]
         # Checking VCS
-        vcs = get_vcs(self._path)
+        vcs = get_vcs(
+            self._original_path if hasattr(self, "_original_path") else self._path
```

Comment:
What we could do is add a `_original_path` to the base `Builder` class which would default to the `value` of `self._path`. That way we avoid this check and always use `self._original_path`.

Reply:
Good idea :+1: